examples/directml/controlnet/README.md

```diff
@@ -0,0 +1,77 @@
+# ControlNet Optimization with DirectML <!-- omit in toc -->
+
+This sample shows how to optimize [Control Net](https://huggingface.co/lllyasviel/sd-controlnet-openpose) with [Stable Diffusion v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5) to run with ONNX Runtime and DirectML.
+
```

Comment:
some of the controlnet scripts that you use from different github, good to put reference from them here - providing the location and mentioning "heavily used the scripts from here modifying to adapt for Olive environment"